docs/includes/trusty-warning.txt

```diff
@@ -0,0 +1,6 @@
+.. warning:: Ubuntu 14.04 has reached End-of-Life on **May 1, 2019** and will
+  no longer receive security updates. If you are still running Ubuntu 14.04
+  on your SecureDrop servers, due to the increased risk of a server compromise,
+  we do not recommend following the procedure outlined below. Instead, we
+  recommend creating a manual backup of submission content, and re-installing 
```

Comment:
"saving old submissions" seems more accurate to me than "creating a manual backup of submission content" - "backup" implies that they can be restored onto the new instance, which isn't the case, and we have an actual backup function that should not be used in this case

Reply:
Done.